.github/workflows/spread-tests.yaml

```
@@ -133,21 +133,30 @@ jobs:
           # Add start line to filtered log
           echo "GRAFANA START: pr ${CHANGE_ID} attempt ${{ github.run_attempt }} run ${{ github.run_id }} group ${{ inputs.group }}" > "$FILTERED_LOG_FILE"
 
-    - name: Download built snap
+    - name: Download built snap (amd64)
       uses: actions/download-artifact@v4
-      if: "!endsWith(inputs.group, '-fips')"
+      if: "!contains(inputs.group, '-arm') && !endsWith(inputs.group, '-fips')"
```

Should this say arm64 explicitly?
```suggestion
      if: "!contains(inputs.group, '-arm64') && !endsWith(inputs.group, '-fips')"
```